FirebaseAdmin/FirebaseAdmin/Auth/UserRecord.cs

```diff
@@ -177,6 +184,30 @@ public string ProviderId
         /// </summary>
         public long TokensValidAfterTimestamp => this.tokensValidAfterTimestamp;
 
+        /// <summary>
+        /// Gets the timestamp representing the time that the user account was created.
+        /// </summary>
+        public long CreatedAt
```

Comment:
This and the next property should be exposed as `UserMetadata`. The master branch already contains the change. Just sync up your branch.